src/pages/workspaces/workspace/analysis/Analyses.test.ts

```diff
@@ -0,0 +1,389 @@
+    // Banner text and images
+    screen.getByText('A place for all your analyses')
+    screen.getByAltText('Jupyter')
+    screen.getByAltText('RStudio Bioconductor')
+    screen.getByAltText('Galaxy')
+  })
+
+  it('loads properly with files for a google workspace', async () => {
+    // Arrange
+    const fileName1 = 'file1.ipynb'
+    const fileName2 = 'file2.ipynb'
+    const files = [getFileFromPath(`test/${fileName1}` as AbsolutePath), getFileFromPath(`test/${fileName2}` as AbsolutePath)]
+
+    asMockedFn(useAnalysisFiles).mockImplementation(() => ({
+      ...defaultUseAnalysisStore,
+      loadedState: { status: 'Ready', state: files }
+    }))
+
+    // Act
+    await act(async () => { // eslint-disable-line require-await
+      render(h(BaseAnalyses, defaultAnalysesProps))
+    })
+
+    // Assert
+    screen.getByText('Start')
+    // Table headings
+    screen.getByText('Application')
+    screen.getByText('Name')
+    screen.getByText('Last Modified')
+    screen.getByPlaceholderText('Search analyses')
+    screen.getByText(fileName1)
+    screen.getByText(fileName2)
+  })
+
+  it('loads properly with files for an azure workspace', async () => {
+    // Arrange
+    const fileName1 = 'file1.ipynb'
+    const fileName2 = 'file2.ipynb'
+    const files = [getFileFromPath(`test/${fileName1}` as AbsolutePath), getFileFromPath(`test/${fileName2}` as AbsolutePath)]
+
+    asMockedFn(useAnalysisFiles).mockImplementation(() => ({
+      ...defaultUseAnalysisStore,
+      loadedState: { status: 'Ready', state: files }
+    }))
+
+    // Act
+    await act(async () => { // eslint-disable-line require-await
+      render(h(BaseAnalyses, { ...defaultAnalysesProps, workspace: defaultAzureWorkspace }))
+    })
+
+    // Assert
+    screen.getByText('Start')
+    // Table headings
+    screen.getByText('Application')
+    screen.getByText('Name')
+    screen.getByText('Last Modified')
+    screen.getByPlaceholderText('Search analyses')
+    screen.getByText(fileName1)
+    screen.getByText(fileName2)
+  })
+
+  it('Sorts analysis table properly', async () => {
+    // Arrange
+    const fileName1 = 'file1.ipynb'
+    const fileName2 = 'file2.ipynb'
+    const file1Date = new Date().getTime()
+    const file2Date = file1Date + 10000
+    const files = [{ ...getFileFromPath(`test/${fileName1}` as AbsolutePath), lastModified: file1Date }, { ...getFileFromPath(`test/${fileName2}` as AbsolutePath), lastModified: file2Date }]
+
+    asMockedFn(useAnalysisFiles).mockImplementation(() => ({
+      ...defaultUseAnalysisStore,
+      loadedState: { status: 'Ready', state: files }
+    }))
+
+    // Act
+    await act(async () => { // eslint-disable-line require-await
+      render(h(BaseAnalyses, defaultAnalysesProps))
+    })
+
+    const analysisRowsBeforeSort: HTMLElement[] = await screen.findAllByRole('row')
+
+    await act(async () => {
+      const nameTableHeader = screen.getByText('Name')
+      await userEvent.click(nameTableHeader)
+    })
```

Comment:
Suggest only using `act` around what's absolutely necessary.

```suggestion
    const nameTableHeader = screen.getByText('Name')
    await act(() => userEvent.click(nameTableHeader))
```